index.js

```diff
@@ -21,6 +21,76 @@ app.get('/account', function(request, response) {
 	response.render('pages/account');
 });
 
+app.get('/profile-info/:userid', (request, response) => {
+  let userId = request.params.userid;
+  let profileInfo1 = {
+    "userid": "01",
+    "username": "Harry Potter",
+    "avatar": "/images/ostrich-profile.jpg",
+  };
+  let profileInfo2 = {
+      "userid": "02",
+      "username": "Dumbledore",
+      "avatar": "/images/ostrich-profile.jpg",
+  };
+
+  switch(userId) {
+      case '01':
+          return response.json(profileInfo1);
+          break;
+      case '02':
+          return response.json(profileInfo2);
+          break;
+      default:
+          return response.json('no user found');
+  };
+})
+
+app.get('/watched-list/:userid', (request, response) => {
+  let userId = request.params.userid;
+
+  let watched01 = {
+    data: ['power rangers', 'justice league']
+  };
+  let watched02 = {
+    data: ['die hard', 'blade runner']
+  };
+
+  switch (userId) {
+    case '01':
+      return response.json(watched01);
+      break;
+    case '02':
+      return response.json(watched02);
+      break;
+    default:
+      return response.json('list not found');
+  };
+});
+
+app.get('/towatch-list/:userid', (request, response) => {
+  let userId = request.params.userid;
+
+  let toWatch1 = {
+    data: ['back to the future2', 'star wars'],
+  };
+  let toWatch2 = {
+    data: ['star trek', 'game of thrones'],
+  };
+
+  switch (userId) {
+    case '01':
+      return response.json(toWatch1);
+      break;
+    case '02':
+      return response.json(toWatch2);
+      break;
+    default:
+      return response.json('list not found');
```

Comment:
If no user is found, you should also respond with a 404 ;)

Reply:
Hi @leeDav  :)

This is something we definitely want to do in the future, however, for now we're trying to keep it as simple as possible and expand upon it later

Reply:
@leeDav Thank you for the suggestion :)